CMakeLists.txt

```diff
@@ -14,15 +14,16 @@ file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
 ExternalProject_Add(yaml_cpp_src
   GIT_REPOSITORY  https://github.com/jbeder/yaml-cpp
   GIT_TAG ${YAML_CPP_TAG}
+  UPDATE_COMMAND ""
```

Comment:
@HannesSommer this was missing, now the package generation works properly (no random copies of the headers and libs installed in the home or whatever was the original build dir)

Reply:
Aha! Great that you found it and thanks for notifying!